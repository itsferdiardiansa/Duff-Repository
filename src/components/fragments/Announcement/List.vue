<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :showLoader="isFetching"
        :rowLoader="2"
        :selectableRows="true"
        :pagination="paginationData"
        @onSelectedRowCallback="handleSelectRow"
      >
        <template #link="props">
          <a :href="props.data.link" target="_blank" class="underline"
            >Selengkapnya</a
          >
        </template>

        <template #status="props">
          <Status :data="props.data.status" />
        </template>

        <template #action="props">
          <ActionButton :data="props.data" />
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, onMounted, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import ActionButton from './ActionButton'
import Status from './Status'

export default {
  components: {
    Table,
    ActionButton,
    Status,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Title',
        accessor: 'title',
        width: '30%',
      },
      { title: 'Content', accessor: 'content' },
      { title: 'Link', accessor: 'link', align: 'center' },
      { title: 'Status', accessor: 'status', width: '9%', align: 'center' },
      { title: 'Expired', accessor: 'expired', align: 'center' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        align: 'center',
      },
    ])

    const getAnnouncement = () => {
      store.dispatch('announcement/fetchData')
    }

    const filteredData = computed(() => (
      store.getters['announcement/getAnnouncement']
    ))

    const paginationData = computed(() => (
      store.getters['announcement/getPagination']
    ))

    const isFetching = computed(() => (
      store.getters['announcement/getFetchStatus']
    ))

    const searchFilter = data => {
      console.log(data, 'List')
    }

    const handleSelectRow = () => {
      // console.log(data)
    }

    onMounted(() => {
      getAnnouncement()
    })

    return {
      isFetching,
      filteredData,
      paginationData,
      tHeaders,
      searchFilter,
      handleSelectRow
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
