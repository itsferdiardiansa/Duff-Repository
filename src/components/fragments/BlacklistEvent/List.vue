<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :showLoader="isFetching"
        :rowLoader="2"
      >
        <template v-slot:blacklist="props">
          <div>
            <h3 class="font-black textr-lg mb-2">{{ props.data.title }}</h3>

            <p>
              blacklist at: <span>{{ props.data.blacklistAt }}</span>
            </p>
            <p>
              reported at: <span>{{ props.data.reportedAt }}</span>
            </p>
          </div>
        </template>

        <template v-slot:action>
          <Button :textBold="true" size="sm" variant="warning">
            <img svg-inline class="w-4 h-4" src="@icon/pencil.svg" />
          </Button>
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import Button from '@common/Button'
import data from '@mock/collections'

export default {
  components: {
    Button,
    Table,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Blacklist Event Name / Desc / TNC',
        accessor: 'blacklist',
        width: '40%',
        align: 'left',
      },
      {
        title: 'Created Date',
        accessor: 'createdDate',
        width: '60%',
        align: 'left',
      },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '5%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.blacklistEvent.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
