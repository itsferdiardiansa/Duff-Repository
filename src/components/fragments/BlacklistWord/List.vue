<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :showLoader="isFetching"
        :rowLoader="2"
      >
        <template v-slot:status="props">
          <Badge
            :textBold="true"
            size="sm"
            variant="danger"
            shape="circle"
            :label="props.data.status"
          />
        </template>

        <template v-slot:blacklist_at> Event Name </template>

        <template v-slot:action>
          <Button :textBold="true" size="sm" variant="warning">
            <img svg-inline class="w-4 h-4" src="@icon/pencil.svg" />
          </Button>
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import Badge from '@common/Badge'
import Button from '@common/Button'
import data from '@mock/collections'

export default {
  components: {
    Button,
    Table,
    Badge,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Word',
        accessor: 'path',
        width: '20%',
        align: 'left',
      },
      {
        title: 'Blacklist at',
        accessor: 'blacklist_at',
        width: '60%',
        align: 'left',
      },
      { title: 'Status', accessor: 'status', width: '5%', align: 'center' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '5%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.url_redirection.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
