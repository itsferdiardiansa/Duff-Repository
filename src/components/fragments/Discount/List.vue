<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :isLoading="isFetching"
        :rowLoader="2"
      >
        <template v-slot:eventName="{ data }">
          <div>
            <a :href="data.link" class="text-yellow-600 text-base">{{
              data.title
            }}</a>

            <p class="text-xs">
              creator: <span>{{ data.creator }}</span>
            </p>
          </div>
        </template>

        <template v-slot:discount="{ data }">
          <div class="">
            <p>
              Percentage: <span>{{ data.discount.percentage }}</span>
            </p>
            <p>
              Fixed: <span class="font-black">{{ data.discount.fixed }}</span>
            </p>
          </div>
        </template>

        <template v-slot:discountAmount="{ data }">
          <p class="font-black">{{ data.discount.amount }}</p>
        </template>

        <template v-slot:action="props">
          <ActionButton :data="props.data" />
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import data from '@mock/collections'
import ActionButton from './ActionButton'

export default {
  components: {
    Table,
    ActionButton,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Title',
        accessor: 'title',
        width: '20%',
        align: 'left',
      },
      {
        title: 'Event Name',
        accessor: 'eventName',
        width: '20%',
        align: 'left',
      },
      { title: 'Start Date', accessor: 'startDate', align: 'center' },
      { title: 'Potential GMV', accessor: 'potentialGMV', align: 'center' },
      { title: 'Discount', accessor: 'discount', align: 'center' },
      {
        title: 'Discount',
        accessor: 'discountAmount',
        width: '10%',
        align: 'center',
      },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.discount.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
