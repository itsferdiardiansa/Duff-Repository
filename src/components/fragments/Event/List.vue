<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :emptyDataComponent="emptyDataComponent"
        :showLoader="isFetching"
        :rowLoader="2"
      >
        <template v-slot:imgThumb="props">
          <div class="flex justify-center">
            <div class="img-thumb w-24 inline-block">
              <img :src="props.data.imgThumb" />
            </div>
          </div>
        </template>

        <template v-slot:event="props">
          <div>
            <div class="title">
              <h3 class="font-black text-lg">
                {{ props.data.title }}
              </h3>
            </div>

            <div class="flex mt-2">
              <div>
                <p>Total ticket sold: <strong>0</strong></p>
                <p>Total paid: <strong>0</strong></p>
              </div>

              <div class="ml-5">
                <p>
                  Event Date: <strong>{{ props.data.eventDate }}</strong>
                </p>
                <p>
                  Event Type: <strong>{{ props.data.eventType }}</strong>
                </p>
              </div>
            </div>
          </div>
        </template>

        <template v-slot:status="props">
          <div class="text-center">
            <Badge :dot="true" variant="success" />
            <span class="ml-2">{{
              props.data.status.tier ? 'Active' : 'Inactive'
            }}</span>
            <span class="px-2">|</span>
            <span>{{ props.data.status.role }}</span>
          </div>
        </template>

        <template v-slot:action="props">
          <ActionButton :data="props.data" />
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import EmptyData from './EmptyData'
import ActionButton from './ActionButton'
import Badge from '@common/Badge'
import data from '@mock/collections'

export default {
  components: {
    Table,
    Badge,
    ActionButton,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Image',
        accessor: 'imgThumb',
      },
      {
        title: 'Event',
        accessor: 'event',
      },
      {
        title: 'Created Date',
        accessor: 'created_date',
      },
      {
        title: 'Status',
        accessor: 'status',
      },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '20%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.event.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    const emptyDataComponent = computed(() => EmptyData)

    return {
      isFetching,
      filteredData,
      tHeaders,
      emptyDataComponent,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
