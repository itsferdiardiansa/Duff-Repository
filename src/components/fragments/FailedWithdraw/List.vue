<template>
  <div class="container">
    <div class="wrapper">
      <Table
        :headers="tHeaders"
        :items="filteredData"
        :isLoading="isFetching"
        :rowLoader="2"
      >
        <template v-slot:event_detail="props">
          <h3 class="font-black text-lg mb-3">{{ props.data.title }}</h3>

          <p>
            Date: <span class="font-bold">{{ props.data.date }}</span>
          </p>
          <p>
            Time: <span class="font-bold">{{ props.data.time }}</span>
          </p>
        </template>

        <template v-slot:organization="props">
          <div class="flex">
            <div class="img-thumb">
              <img :src="props.data.imgThumb" class="w-20" />
            </div>

            <div class="ml-3">
              <h3 class="font-black text-lg mb-2">
                {{ props.data.organizationName }}
              </h3>

              <p class="text-xs mb-1">{{ props.data.email }}</p>
              <p class="text-xs">{{ props.data.phone }}</p>
            </div>
          </div>
        </template>

        <template v-slot:widthdraw_detail="props">
          <p>
            Requested Date:
            <span class="font-bold">{{ props.data.withdrawDetail.date }}</span>
          </p>
          <p>
            Withdraw Amount:
            <span class="font-bold">{{
              props.data.withdrawDetail.amount
            }}</span>
          </p>
          <p>
            Error Message:
            <span class="font-bold">{{
              props.data.withdrawDetail.errorMsg
            }}</span>
          </p>
        </template>

        <template v-slot:action>
          <Button :textBold="true" size="sm" variant="dark">
            <img svg-inline class="w-4 h-4" src="@icon/pencil.svg" />
          </Button>
        </template>
      </Table>
    </div>
  </div>
</template>
<script>
import { computed, ref } from 'vue'
import { useStore } from 'vuex'
import Table from '@common/Table'
import Button from '@common/Button'
import data from '@mock/collections'

export default {
  components: {
    Button,
    Table,
  },
  setup() {
    const store = useStore()

    let tHeaders = ref([
      {
        title: 'Event Detail',
        accessor: 'event_detail',
        width: '30%',
        align: 'left',
      },
      { title: 'Organization', accessor: 'organization', align: 'left' },
      { title: 'Withdraw Detail', accessor: 'widthdraw_detail', align: 'left' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 3,
        width: '5%',
        align: 'center',
      },
    ])

    const filteredData = computed(() => {
      return data.failedWithdraw.result
    })

    const isFetching = computed(() => {
      return store.getters['thematicPage/getFetchStatus']
    })

    return {
      isFetching,
      filteredData,
      tHeaders,
    }
  },
}
</script>
<style lang="scss" scoped>
.container {
  @apply relative;

  .wrapper {
    @apply relative;
  }
}
</style>
