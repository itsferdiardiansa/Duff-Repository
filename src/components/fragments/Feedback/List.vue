<template>
  <div class="list">
    <Table
      :headers="tHeaders"
      :items="filteredData"
      :isFetching="requestStatus.fetch"
      :onError="requestStatus.error.status"
      :pagination="pagination"
      :onPageChange="handlePageChange"
      :onFailedFetchHandler="fetchData"
    >
      <template #read_by_slice="{ data }">
        <div class="read-by-col">
          <template v-for="(item, key) in data.read_by_slice" :key="key">
            <Badge variant="primary" :label="item" />
          </template>
        </div>
      </template>

      <template #responded_by="{ data }">
        <Badge variant="primary" :label="data.responded_by" />
      </template>

      <template #action="{ data }">
        <ActionButton :data="actionButtons" :item="data" />
      </template>
    </Table>
  </div>
</template>
<script>
import { computed, onMounted, reactive, ref } from 'vue';
import { useStore } from 'vuex';
import Table, { ActionButton } from '@common/Table';
import Badge from '@common/Badge';

export default {
  components: {
    Table,
    Badge,
    ActionButton,
  },
  setup() {
    const store = useStore();
    const params = reactive({ page: 1, limit: 5 });

    const tHeaders = ref([
      {
        title: 'Email',
        accessor: 'email',
        width: '15%',
        align: 'left',
      },
      { title: 'Name', accessor: 'name', width: '15%' },
      { title: 'Page', accessor: 'page' },
      { title: 'Section', accessor: 'section' },
      { title: 'Responded by', accessor: 'responded_by' },
      { title: 'Read by', accessor: 'read_by_slice' },
      {
        title: 'Action',
        accessor: 'action',
        colSpan: 1,
        align: 'center',
      },
    ]);

    const actionButtons = reactive([
      {
        variant: 'dark',
        icon: ['fa', 'marker'],
        text: 'Mark as responded',
        bold: true,
        onClickFn: (e, data) => {
          markAsResponded(e, data);
        },
      },
      {
        variant: 'dark',
        icon: ['fa', 'bookmark'],
        text: 'Mark as read',
        inverse: true,
        bold: true,
        onClickFn: (e, data) => {
          markAsRead(e, data);
        },
      },
    ]);

    const fetchData = () => {
      store.dispatch('feedback/fetchData', params);
    };

    const markAsResponded = (e, data) => {
      store.dispatch('feedback/markAsResponded', {
        action: 'form.create',
        data,
        params,
      });
    };

    const markAsRead = (e, data) => {
      store.dispatch('feedback/markAsRead', {
        action: 'form.create',
        data,
        params,
      });
    };

    const filteredData = computed(() => {
      return store.getters['feedback/getItems'];
    });

    const pagination = computed(() => {
      return store.getters['feedback/getPagination'];
    });

    const requestStatus = computed(() => {
      return store.getters['feedback/getRequestStatus'];
    });

    const handlePageChange = pageParams => {
      Object.assign(params, pageParams);

      fetchData();
    };

    onMounted(fetchData);

    return {
      fetchData,
      requestStatus,
      filteredData,
      tHeaders,
      actionButtons,
      pagination,
      handlePageChange,
    };
  },
};
</script>
<style lang="scss" scoped>
.list {
  @apply relative;

  .wrapper {
    @apply relative;
  }
  .read-by-col {
    @apply flex flex-wrap justify-center;
    max-width: 200px;

    span {
      @apply mt-1 mr-1;
    }
  }
}
</style>
