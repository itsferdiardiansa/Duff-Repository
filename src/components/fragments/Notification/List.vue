<template>
  <div class="list">
    <Modal
      title="Delete confirmation"
      description="Are you sure you want to delete this item?"
      :onConfirmFn="deleteData"
    />

    <Table
      :headers="tHeaders"
      :items="filteredData"
      :isFetching="requestStatus.fetch"
      :onError="requestStatus.error.status"
      :onFailedFetchHandler="fetchData"
      :pagination="pagination"
      :onPageChange="handlePageChange"
    >
      <template #description="{ data }">
        <div class="description-col">
          <h3 style="word-break: break-word">
            {{ data.title }}
          </h3>

          <p class="article" v-html="data.description"></p>

          <div>
            <a :href="data.link" class="link" target="_blank">Selengkapnya</a>
          </div>
        </div>
      </template>

      <template #action="{ data }">
        <ActionButton :data="actionButtons" :item="data" />
      </template>
    </Table>
  </div>
</template>
<script>
/* eslint-disable */
import { computed, onMounted, reactive, ref } from 'vue';
import { useStore } from 'vuex';
import Table, {
  EmptyTable as emptyComponent,
  ActionButton,
} from '@common/Table';
import Badge from '@common/Badge';
import Modal from '@common/Modal';
import { useRouter } from 'vue-router';

export default {
  components: {
    Table,
    Badge,
    Modal,
    ActionButton,
  },
  setup() {
    const store = useStore();
    const router = useRouter();
    const params = reactive({ page: 1, limit: 5 });

    const tHeaders = ref([
      { title: 'Message', width: '80%', accessor: 'description' },
      {
        title: 'Action',
        accessor: 'action',
        align: 'center',
      },
    ]);

    const actionButtons = ref([
      {
        icon: ['fa', 'pencil-alt'],
        variant: 'dark',
        onClickFn: (e, data) => {
          router.push({
            name: 'Update Notification',
            params: {
              hash_id: data.hash_id,
              data: JSON.stringify(data),
            },
          });
        },
      },
      {
        icon: ['fa', 'trash'],
        variant: 'dark',
        onClickFn: (e, data) => {
          toggleModal(e, data);
        },
      },
    ]);

    const toggleModal = (e, data) => {
      self.$modal.show(data);
    };

    const fetchData = () => {
      store.dispatch('notification/fetchData', params);
    };

    const pagination = computed(() => {
      return store.getters['notification/getPagination'];
    });

    const filteredData = computed(() => {
      return store.getters['notification/getItems'];
    });

    const requestStatus = computed(() => {
      return store.getters['notification/getRequestStatus'];
    });

    const deleteData = ({ hash_id }) => {
      store.dispatch('notification/deleteData', {
        action: 'form.delete',
        hash_id,
        params,
      });
    };

    const handlePageChange = pageParams => {
      Object.assign(params, pageParams);

      fetchData();
    };

    onMounted(fetchData);

    return {
      requestStatus,
      filteredData,
      tHeaders,
      fetchData,
      deleteData,
      emptyComponent,
      actionButtons,
      toggleModal,
      pagination,
      handlePageChange,
    };
  },
};
</script>
<style lang="scss" scoped>
.list {
  @apply relative;

  .wrapper {
    @apply relative;
  }

  .previlage-col {
    @apply flex justify-center;
  }

  .description-col {
    @apply text-left;

    h3 {
      @apply text-base font-bold max-w-xl;
    }

    .article {
      @apply text-left mt-1 text-sm;
    }

    .link {
      @apply text-blue-500;
    }
  }
}
</style>
