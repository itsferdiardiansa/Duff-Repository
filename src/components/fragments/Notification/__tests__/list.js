import { mount } from '@vue/test-utils';
import { createStore } from 'vuex';
import List from '@fragment/Notification/List';
import notificationModule from '@store/modules/notification';

describe('fragment/Notification/List', () => {
  let store;

  beforeAll(() => {
    store = createStore({
      modules: {
        notification: notificationModule,
      },
    });
  });

  it('will render and match snapshot', async () => {
    const Component = mount(<List />, {
      global: {
        plugins: [store],
      },
    });

    await Component.vm.$store.dispatch('notification/fetchData', {
      page: 1,
      limit: 10,
    });
    expect(Component.html()).toMatchSnapshot();
  });
});
