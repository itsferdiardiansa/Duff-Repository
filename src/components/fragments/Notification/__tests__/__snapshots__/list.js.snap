// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fragment/Notification/List will render and match snapshot 1`] = `
<div class="list">
  <!--v-if-->
  <!--v-if-->
  <table class="ss-table">
    <thead class="ss-table--head" headers="[object Object],[object Object]" items="[object Object],[object Object],[object Object]" isfetching="false" pagination="[object Object]" emptymessage="Empty data" rownumber="true" rowloader="5" selectablerows="false" withpagination="true" withfilter="false" filteroptions="">
      <tr>
        <th width="4%">
          <!--v-if--><span>#</span>
        </th>
        <th class="" width="80%">Message</th>
        <th class="" align="center">Action</th>
      </tr>
    </thead>
    <tbody class="ss-table--body">
      <!--v-if-->
      <tr class="table-content--body-row">
        <td class="ss-table--body-col"><label>1</label></td>
        <td class="ss-table--body-col">
          <div class="description-col">
            <h3 style="word-break: break-word;"></h3>
            <p class="article">
            <p>Text</p>
            </p>
            <div><a class="link" target="_blank">Selengkapnya</a></div>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper"><button class="ss-button ss-button--dark ss-button--base">
                <div class="ss-button--wrapper">
                  <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 512 512">
                      <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                    </svg></div>
                  <!--v-if-->
                </div>
              </button><button class="ss-button ss-button--dark ss-button--base">
                <div class="ss-button--wrapper">
                  <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 448 512">
                      <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                    </svg></div>
                  <!--v-if-->
                </div>
              </button></div>
          </div>
        </td>
      </tr>
      <tr class="table-content--body-row">
        <td class="ss-table--body-col"><label>2</label></td>
        <td class="ss-table--body-col">
          <div class="description-col">
            <h3 style="word-break: break-word;"></h3>
            <p class="article">1</p>
            <div><a class="link" target="_blank">Selengkapnya</a></div>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper"><button class="ss-button ss-button--dark ss-button--base">
                <div class="ss-button--wrapper">
                  <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 512 512">
                      <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                    </svg></div>
                  <!--v-if-->
                </div>
              </button><button class="ss-button ss-button--dark ss-button--base">
                <div class="ss-button--wrapper">
                  <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 448 512">
                      <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                    </svg></div>
                  <!--v-if-->
                </div>
              </button></div>
          </div>
        </td>
      </tr>
      <tr class="table-content--body-row">
        <td class="ss-table--body-col"><label>3</label></td>
        <td class="ss-table--body-col">
          <div class="description-col">
            <h3 style="word-break: break-word;"></h3>
            <p class="article">Deskripsi partner</p>
            <div><a class="link" target="_blank">Selengkapnya</a></div>
          </div>
        </td>
        <td class="ss-table--body-col text-center">
          <div class="ss-tb--action">
            <div class="wrapper"><button class="ss-button ss-button--dark ss-button--base">
                <div class="ss-button--wrapper">
                  <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 512 512">
                      <path fill="currentColor" d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"></path>
                    </svg></div>
                  <!--v-if-->
                </div>
              </button><button class="ss-button ss-button--dark ss-button--base">
                <div class="ss-button--wrapper">
                  <div class="ss-button--icon"><svg xmlns="http://www.w3.org/2000/svg" class="" viewBox="0 0 448 512">
                      <path fill="currentColor" d="M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z"></path>
                    </svg></div>
                  <!--v-if-->
                </div>
              </button></div>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
  <div class="ss-pagination">
    <div class="ss-pagination--info">
      <p> Showing <span>1</span> to <span>10</span> of <span>10</span> results </p>
    </div>
    <div class="ss-pagination--page"><span class="page-number active">1</span></div>
  </div>
</div>
`;
